voltixApp/VoltixApp/Model/Coin.swift

```diff
@@ -95,7 +96,12 @@ class Coin: Codable, Hashable {
     }
     
     func getMaxValue(_ fee: BigInt) -> Decimal {
-        let maxValue = (BigInt(rawBalance, radix: 10) ?? 0) - fee
+        
+        // USDC has 6 decimals
```

Comment:
I believe this is not generic , and won't work for other coins , for example 
BTC is using 1E8 
ATOM is using 1E6

Reply:
UTXO does not use this, only EVM

Reply:
then , maybe rename it to something more EVM specific , for example: `getEVMMaxValue`

Reply:
Thanks for your catch. I am fixing it.

Reply:
Fixed and tested on:
Avax
USDC-avax
Thor RUNE
Sol